bootstrap/yarn.lock

```diff
@@ -1085,13 +1085,31 @@
     rxjs "^5.5.6"
     uuid "^3.2.1"
 
-"@gooddata/goodstrap@^68.1.0":
-  version "68.2.1"
-  resolved "https://registry.yarnpkg.com/@gooddata/goodstrap/-/goodstrap-68.2.1.tgz#da8c6eadb257d23dc1757c25c33279e1d4a1b1da"
-  integrity sha512-qZQpQSrPqoir4GDSE/WtnoXWJNUwm4TnHwWWhm9lqTnQy5NliyCiMDO6Ro942lRozy+D0S4+eR37OktNFdjO8Q==
+"@gooddata/gooddata-js@^12.5.4":
```

Comment:
Please run `yarn-deduplicate --packages "@gooddata/gooddata-js"` so that there is only one version here.

Reply:
Fixed. Thanks